src/pspm_trim.m

```diff
@@ -263,65 +271,73 @@
         sta_offset = 0;
     end
 end
+sta_time = sta_p + sta_offset;
 if (sto_p + sto_offset) > infos.duration
     warning('ID:marker_out_of_range', ['\nEnd point (%.2f s) outside ', ...
         'file, no trimming at end.'], (sto_p + sto_offset));
-    if (sto_p > infos.duration)
-        sto_p = infos.duration;
-        sto_offset = 0;
-    else
-        sto_offset = infos.duration - sto_p;
-    end
+    % adjustment of the end point is being taken care of in pspm_epochs2logical
+    sto_time = infos.duration - sta_p - sta_offset;
```

Comment:
* If `sto_time` is a duration
  * When `(sto_p + sto_offset) <= infos.duration`, why `sto_time = sto_p + sto_offset` instead of `sto_time = sto_p + sto_offset - sta_p - sta_offset`, given `sto_time = infos.duration - sta_p - sta_offset` if `(sto_p + sto_offset) > infos.duration`?

* If `sto_time` is a time point
  * Should not `sto_time` be equal to `infos.duration` if `(sto_p + sto_offset)` exceeds `infos.duration`?

Reply:
1. `sto_time` is never a duration.
2. Correct; fixed.